Sources/MapboxNavigation/Constants.swift

```diff
@@ -46,6 +46,11 @@ public let NavigationViewMinimumVolumeForWarning: Float = 0.3
  */
 public var GradientCongestionFadingDistance: CLLocationDistance = 30.0
 
+/**
+ The maximum distance threshold of vanishing route line update. When the user's location to the route line is larger than the threshold, the user is off the route and the route line won't be updated.
+ */
+public var OffRouteDistanceUpdateThreshold: CLLocationDistance = 15.0
```

Comment:
To play it safe I'd rather keep this value internal for now. We'll have a chance to do a couple rounds of testing first and then see whether this is really something we should expose as public variable.

Reply:
Thanks for the suggestion, I'll make this value internal now.